src/render/mod.rs

```diff
@@ -370,10 +370,10 @@ impl Renderer {
                             0,
                         );
                     } else {
-                        debug!("1: not rendering solid {:?}", pos);
+                        trace!("1: not rendering solid {:?}", pos);
```

Comment:
Could you remove those

Reply:
is that the only one?

Reply:
No, all the debugs you changed to trace

Reply:
okay! 
also the one in world/mod.rs?

Reply:
yea

Reply:
finish. 
dropped the other commit "debug -> trace"  and add another commit where I remove debug instead